libnavigation-base/src/main/java/com/mapbox/navigation/base/route/NavigationRoute.kt

```diff
@@ -186,12 +186,7 @@ class NavigationRoute internal constructor(
             logI("NavigationRoute.createAsync is called", LOG_CATEGORY)
             return coroutineScope {
                 val deferredResponseParsing = async(ThreadController.DefaultDispatcher) {
-                    DirectionsResponse.fromJson(directionsResponseJson.asReader()).also {
-                        logD(
-                            "parsed directions response to java model for ${it.uuid()}",
-                            LOG_CATEGORY
-                        )
```

Comment:
maybe it's worth leaving logs? As I remember we added them in case of performance complains that could be reproduced only on customer's machines

Reply:
Oops, I removed them by accident.